e2e-tests/production-runtime/cypress/integration/resource-loading-resilience.js

```diff
@@ -2,6 +2,12 @@ const waitForAPIOptions = {
   timeout: 5000,
 }
 
+Cypress.on('uncaught:exception', () => {
```

Comment:
https://docs.cypress.io/guides/references/migration-guide#Uncaught-exception-and-unhandled-rejections